src/main/java/cristiano/Ronaldo.java

```diff
@@ -0,0 +1,110 @@
+package cristiano;
+
+import java.util.ArrayList;
+
+/**
+ * This class includes the main methods to handle inputs from the user.
+ * The methods include special methods such as "boast" and "exclaim"
+ * which represents the chatbot's personality.
+ * Each task is represented as a goal, which includes a goal number and completion.
+ */
+public class Ronaldo {
+    private static final ArrayList<Goal> goals = new ArrayList<>();
+
+    public void greet() {
+        String indent = "------------------------------------------------------------------------------ \n";
```

Comment:
Perhaps we can declare this as a constant variable just like this:
`private static final String INDENT = "____________________________________________________________";`

Reply:
Hi, thanks for your feedback! I appreciate it a lot.